DigitalLearningSolutions.Web/ViewModels/LearningPortal/Current/CurrentLearningItemViewModel.cs

```diff
@@ -19,15 +20,16 @@ ReturnPageQuery returnPageQuery
         public DateTime? CompleteByDate { get; }
         public OldDateValidator.ValidationResult? CompleteByValidationResult { get; set; }
         public ReturnPageQuery ReturnPageQuery { get; }
+        private readonly IClockUtility clockUtility = new ClockUtility();

```

Comment:
Ideally we'd just pass in the current time, but this model is constructed by a lot of different services / controllers, so it's probably not worth it. Let's just leave it as is.